ocaml/xapi-idl/cluster/cluster_interface.ml

```diff
@@ -79,6 +69,18 @@ let decode_cluster_config x =
 type cluster_config_and_all_members = cluster_config * all_members
 [@@deriving rpcty]
 
+type tls_config = {
+    server_pem_path: string  (** Path containing private and public keys *)
+  ; cn: string
+        (** CN used for verification, typically host UUID, ignored otherwise *)
```

Comment:
If this is shared among the cluster shouldn't this real pool UUID instead of host UUID?

Reply:
This is misleading. We are using something static because this is a pool-level configuration. Currently something like "api-cluster" or so. I have to look it up. Should update the comment.